doc/Examples/01-passwords.php

```diff
@@ -13,7 +13,7 @@
 }
 
 $password = new HiddenString('correct horse battery staple');
-$hash = Password::hash($password, $secretKey);
+$hash     = Password::hash($password, $secretKey);
```

Comment:
Please don't align these as they cause horrible diffs, phpdoc doesn't change that often and therefor is exceptional for this alignment.